quickwit-cli/src/lib.rs

```diff
@@ -211,7 +211,11 @@ pub async fn extract_split_cli(args: ExtractSplitArgs) -> anyhow::Result<()> {
     let split_file = PathBuf::from(format!("{}.split", args.split_id));
     let (_, bundle_footer) = read_split_footer(index_storage.clone(), &split_file).await?;
 
-    let bundle_storage = BundleStorage::new(index_storage, split_file, &bundle_footer)?;
+    let (_hotcache_bytes, bundle_storage) = BundleStorage::open_from_split_data_with_owned_bytes(
```

Comment:
There is something wrong here. The docs of `open_from_split_data_with_owned_bytes` indicates 
```
/// The provided data must include the footer_bytes at the end of the slice, but it can have
/// more up front.
```
What is the `footer_bytes`? Should it include the hotcache or not? By reading the code, we see that this method call `open_from_split_data` which returns `(Hotcache, BundleStorage)` so now we understand what it returns.

But... `read_split_footer` returns `(split_footer, only_bundle_footer)` so this can't work.

It seems that we can remove `open_from_split_data_with_owned_bytes`. Can you update the docs too? 

The docs of `read_split_footer` is not very clear too: `/// Loads the split footer from a storage and path.` but it returns 2 owned bytes `split_footer, only_bundle_footer`, I need to read the code to understand what it is exactly.